### ISSUE TITLE
coroutine 'noop' was never awaited

### ISSUE THREAD

**Issue by AmericanY** [aio-libs/aiohttp#5394]
I keep getting the following error:

```
C:\Users\AmericaN\Desktop\Lab\MyEnv\lib\site-packages\aiohttp\client.py:977: RuntimeWarning: 
coroutine 'noop' was never awaited
  self._resp.release()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```

**Comment by Dreamsorcerer** [comment]
Given that there is no aiohttp code in your example, I'd suggest filing a bug report to arsenic? Which appears to be using aiohttp.

**Comment by Harmon758** [comment]
Those are warnings, not errors.
What version of aiohttp are you using?
This should be fixed with #4322 and #4333 in v3.7.0 and newer.
See #4282.

**Comment by AmericanY** [comment]
@Harmon758 yes, just removed `4.0.0a1` and installed `3.7.3`. issue resolved. thanks a lot
